src/Core/Plugin.php

```diff
@@ -170,6 +184,24 @@ public static function load($plugin, array $config = [])
     }
 
     /**
+     * Loads the config mappings for plugins.
+     *
+     * The map data in CONFIG/plugins.php map data is used set default paths
+     * for where plugins are located.
+     *
+     * @return void
+     */
+    protected static function _loadConfigMap()
+    {
+        if (!file_exists(CONFIG . 'plugins.php')) {
+            static::$_config = [];
+            return;
+        }
+        $config = include CONFIG . 'plugins.php';
```

Comment:
`Configure::load` does seem a little better. 

The only thing I don't like about this is we're storing a config file for "this name is in this location" and `composer.json` also does "this name(space) is in this location" - but to a certain extent it becomes a choice of which is the lesser evil.